static/_redirects

```diff
@@ -469,9 +469,13 @@
 /docs/setup/version-skew-policy/     /docs/setup/release/version-skew-policy/   301
 
 /docs/setup/minikube/     /docs/tasks/tools/ 302
+/zh/docs/setup/minikube/     /zh/docs/tasks/tools/ 302
 /docs/setup/learning-environment/     /docs/tasks/tools/  302!
+/zh/docs/setup/learning-environment/     /zh/docs/tasks/tools/  302!
 /docs/setup/learning-environment/kind/  /docs/tasks/tools/  302
+/zh/docs/setup/learning-environment/kind/  /zh/docs/tasks/tools/  302
```

Comment:
I don't think this is necessary.
We don't have the kind page or the minikube page.

line 478 and 472 are useless too.
It is terrible to have 15+ redirection entries for each localization.
Where are we heading?